packages/now-cli/test/dev/integration.js

```diff
@@ -809,3 +824,62 @@ test('[now dev] render warning for empty cwd dir', async t => {
     dev.kill('SIGTERM');
   }
 });
+
+test('[now dev] do not rebuild for changes in the output directory', async t => {
+  const directory = fixture('output-is-source');
+
+  // Pack the builder and set it in the now.json
+  const builder = await getPackedBuilderPath('now-static-build');
+
+  await fs.writeFile(
+    path.join(directory, 'now.json'),
+    JSON.stringify({
+      builds: [
+        {
+          src: 'package.json',
+          use: `file://${builder}`,
```

Comment:
I didn't even know that file urls would work 😮

Reply:
I guess it will only work for `now dev` which is fine because this test is for `now dev` 😄